openhtf/__init__.py

```diff
@@ -37,11 +38,35 @@
 
 FLAGS = gflags.FLAGS
 FLAGS(sys.argv)
-
+gflags.DEFINE_string('log_dir', '/tmp', 'dir for log files')
+gflags.DEFINE_enum('log_output', 'console', ['console','file','both'], 'output option')
+gflags.DEFINE_enum('level', 'info', ['debug', 'info', 'warning', 'error', 'critical'], 'log level')
+
+def Setup_logger():
+    htflogger = logging.getLogger('HTFTest')
+    htflogger.setLevel(FLAGS.level.upper())
+    htflogger.propagate = False
+    formatter = logging.Formatter('%(asctime)s - %(levelname)s - %(message)s')
+
+    cur_time = datetime.utcnow().strftime('%Y-%m-%d-%H:%M:%S.%f')[:-3]
+    fh = logging.FileHandler(FLAGS.log_dir + '/test_' + cur_time +'.log', delay=True)
+    fh.setFormatter(formatter)
+    htflogger.addHandler(fh)
+
+    sh = logging.StreamHandler()
+    sh.setFormatter(formatter)
+    htflogger.addHandler(sh)
+
```

Comment:
Add test_record handler (is currently in util/htflogger, but probably move it to test_record).